vnext/Microsoft.ReactNative.Cxx/JSValue.h

```diff
@@ -62,6 +62,13 @@ struct JSValueObject : std::map<std::string, JSValue, std::less<>> {
 
 #pragma endregion
 
+#pragma region Change JSValueObject
+  //! Get a reference to object property value if the property is found,
+  //! or a reference to a new property created with JSValue::Null value otherwise.
+  JSValue &operator[](std::string_view propertyName) noexcept;
+
```

Comment:
could you add a const version too?
const  JSValue &operator[](std::string_view propertyName) const noexcept;

Reply:
It is already there. :) It was the source of the issue because it hides the base class mutating implementation. Please see the PR description.

Reply:
ah sorry missed that, thanks!

---
In reply to: [391317968](https://github.com/microsoft/react-native-windows/pull/4294#discussion_r391317968) [](ancestors = 391317968)